src/test/resources/commit.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "559aa7ba386219254f9448ed24cdaa6e914e5fde",
-  "displayId": "559aa7ba386",
+  "displayId": "feature/myfeature",
```

Comment:
Thanks for doing this. This isn't the first time we've had issues with slashes in paths, so glad to see more automated testing for this.